Classes/ScrnPlayerController.uc

```diff
@@ -1487,6 +1487,33 @@ function ServerLockWeapons(bool bLock)
     bWeaponsLocked = Mut.bAllowWeaponLock && bLock;
 }
 
+//handles changing weapon's inventorygroup
+exec function MoveWepToSlot(int Slot)
+{
+    local KFWeapon weap;
+    local String s; //for console printout
+
+    if (Pawn == none || Pawn.Weapon == none)
+        return;
+
+    weap = KFWeapon(Pawn.Weapon);
+    
+
+    if (weap== none )
+        return;
+    
+    if (Slot < 1 || Slot > 5)
```

Comment:
I suggest removing the upper limit. It will allow the player to define extra key bindings:
```
set input SwitchWeapon 6
GunSlot 6
```

```suggestion
    if (Slot < 1)
```